test/jobs/close_patients_job_test.rb

```diff
@@ -156,7 +160,7 @@ def teardown
     patient.jurisdiction.update(send_close: false)
     ClosePatientsJob.perform_now
     assert_equal(ActionMailer::Base.deliveries.count, 1)
-    assert_histories_contain(patient, 'Monitoree has completed monitoring.')
+    assert_histories_contain(patient, 'Monitoree has completed monitoring. Reason: Enrolled more than 14 days after last date of exposure (system)')
```

Comment:
Shouldn't `14` here be replaced with `ADMIN_OPTIONS['monitoring_period_days']`?

Reply:
Just pushed a change to address this https://github.com/SaraAlert/SaraAlert/pull/1014/commits/97ac96bd1a263583d016c9aa8e13452b7dc31cde